css/src/components/table.scss

```diff
@@ -0,0 +1,67 @@
+$table-cell-padding-xxs: $spacer-4 !default;
+$table-cell-padding-xs: $spacer-5 !default;
+
+$table-caption-size: $font-size-9 !default;
+$table-caption-spacing: 0.125rem !default;
+
+.table-wrapper {
+	overflow-x: auto;
+	overflow-y: hidden;
+	-webkit-overflow-scrolling: touch;
+}
+
+.table {
+	width: 100%;
+	font-size: $font-size-8;
+	border-collapse: collapse;
+	table-layout: auto;
+
+	caption {
+		padding: $spacer-6;
```

Comment:
Should `$spacer-6` be placed into a variable too?

Reply:
Updated